lua/cfc_ulx_commands/curse/effects/respawn.lua

```diff
@@ -0,0 +1,20 @@
+local EFFECT_NAME = "Respawn"
+
+CFCUlxCurse.RegisterEffect( {
+    name = EFFECT_NAME,
+
+    onStart = function( cursedPly )
+        if CLIENT then return end
+
+        cursedPly:Spawn()
```

Comment:
[KillSilent](https://wiki.facepunch.com/gmod/Player:KillSilent) might me more evil as they'll actually die, could run a timer that randomly triggers it too

Reply:
I feel like it's more cursed to just randomly get teleported to spawn with no indication of why, especially since `:KillSilent()` causes a specific death 'animation' to play out that isn't seen anywhere else and thus makes it very identifiable.

You did give me an idea for another type of effect to add, though...